app/controllers/bonus_engine/api/admin/events_controller.rb

```diff
@@ -38,7 +38,7 @@ def find_cycle
         end
 
         def event_params
-          params.require(:event).permit(:name, :opens_at, :closes_at)
+          params.require(:event).permit(:name, :opens_at, :closes_at, :budget, :maximum_points, :msg_required, :minimum_people)
```

Comment:
this line is too long xD. At least, please split it into smaller ones (legilibilitity or however it spells); also it could be useful to have a permitted attributes helper where you can define it ... but that's too much for this right now ^^